packages/fhicl-cpp/package.py

```diff
@@ -78,8 +78,11 @@ def setup_build_environment(self, env):
         sanitize_environments(env, "PATH")
 
     def setup_run_environment(self, env):
-        bindir = self.prefix.bin
+        env.prepend_path("PATH", self.prefix.bin)
+        env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+        env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
`fhicl-cpp` should not need to adjust `ROOT_INCLUDE_PATH` and `CET_PLUGIN_PATH`--I'd remove these two lines.